content/api/metrics.apib

```diff
@@ -1348,6 +1348,121 @@ in the world.
         }
 
 
+### Metrics by Mailbox Provider [GET /v1/metrics/deliverability/mailbox-provider{?from,to,delimiter,domains,campaigns,templates,sending_ips,ip_pools,sending_domains,subaccounts,precision,metrics,timezone,limit,order_by}]
+
+Provides aggregate metrics grouped by mailbox provider over the time window specified.
+
++ Parameters
+    + from (required, datetime, `2021-03-02T08:00`) ... Datetime in format of `YYYY-MM-DDTHH:MM`.
+    + to = `now` (optional, datetime) ... Datetime in format of `YYYY-MM-DDTHH:MM`.
+    + delimiter = `,` (optional, string) ... Specifies the delimiter for query parameter lists.
+    + domains (optional, list) ... Delimited list of recipient domains to include.
+    + campaigns (optional, list) ... Delimited list of campaigns to include.
+    + templates (optional, list) ... Delimited list of template IDs to include.
+    + sending_ips (optional, list) ... Delimited list of sending IPs to include.
+    + ip_pools (optional, list) ... Delimited list of IP pools to include.
+    + sending_domains (optional, list) ... Delimited list of sending domains to include.
+    + subaccounts (optional, list) ... Delimited list of subaccount IDs to include.
+    + precision = `1min` (optional, enum) ...
+
+        Precision of the time window (`from`, `to`) bounds. All values valid up to a 24 hour window, except for `hour` which is also valid up a 31 day window.
+        See: [Precision Parameter](#header-precision-parameter).
+
+        + Values
+            + `1min`
+            + `5min`
+            + `15min`
+            + `hour`
+
+    + metrics (required, list) ... Delimited list of metrics for filtering.
+
+        + Values
+            + `count_injected`
+            + `count_bounce`
+            + `count_rejected`
+            + `count_delivered`
+            + `count_delivered_first`
+            + `count_delivered_subsequent`
+            + `total_delivery_time_first`
+            + `total_delivery_time_subsequent`
+            + `total_msg_volume`
+            + `count_policy_rejection`
+            + `count_generation_rejection`
+            + `count_generation_failed`
+            + `count_inband_bounce`
+            + `count_outofband_bounce`
+            + `count_soft_bounce`
+            + `count_hard_bounce`
+            + `count_block_bounce`
+            + `count_admin_bounce`
+            + `count_undetermined_bounce`
+            + `count_delayed`
+            + `count_delayed_first`
+            + `count_rendered`
+            + `count_unique_rendered`
+            + `count_unique_confirmed_opened`
+            + `count_clicked`
+            + `count_unique_clicked`
+            + `count_targeted`
+            + `count_sent`
+            + `count_accepted`
+            + `count_spam_complaint`
+
+    + timezone =`UTC` (optional, string) ... Standard timezone identification string.
```

Comment:
Should include a note about timezone being forced to UTC for day. See https://github.com/SparkPost/developers.sparkpost.com/blob/6a550a11ade6ea34d336ff07545d83281c35471a/content/api/metrics.apib#L535